app/controllers/users/confirmations_controller.rb

```diff
@@ -8,6 +8,7 @@ def show
     if resource.errors.empty?
       notify_school_primary_contact
       sign_in(resource)
+      @current_user = current_user
```

Comment:
@current_user is definitely not necessary. Why can't we just call current_user whereever this is being used?